docs/standard/data/sqlite/custom-versions.md

```diff
@@ -41,14 +38,51 @@ Install-Package SQLitePCLRaw.bundle_e_sqlcipher
 
 ---
 
-## SQLitePCLRaw providers
+## SQLitePCLRaw available providers
+
+When not relying on a bundle, you can use the available providers of SQLite with the core assembly.
+
+| Provider | Description |
+|--|--|
+| [SQLitePCLRaw.provider.dynamic](https://www.nuget.org/packages/SQLitePCLRaw.provider.dynamic) | The `dynamic` provider loads the native library instead of using <xref:System.Runtime.InteropServices.DllImportAttribute?displayProperty=nameWithType> attributes. For more information on using this provider, see [`SQLitePCLRaw` custom providers](#sqlitepclraw-custom-providers). |
+| [SQLitePCLRaw.provider.e_sqlite3](https://www.nuget.org/packages/SQLitePCLRaw.provider.e_sqlite3) | The `e_sqlite3` is the default provider. |
+| [SQLitePCLRaw.provider.e_sqlcipher](https://www.nuget.org/packages/SQLitePCLRaw.provider.e_sqlcipher) | The `e_sqlcipher` provider is the unofficial and unsupported `SQLCipher`. |
+| [SQLitePCLRaw.provider.sqlite3](https://www.nuget.org/packages/SQLitePCLRaw.provider.sqlite3) | The `sqlite3` provider is a system-provided `SQLite` for iOS, macOS, and Linux. |
+| [SQLitePCLRaw.provider.sqlcipher](https://www.nuget.org/packages/SQLitePCLRaw.provider.sqlcipher) | The `sqlcipher` provider is for official `SQLCipher` builds from `Zetetic`. |
+| [SQLitePCLRaw.provider.winsqlite3](https://www.nuget.org/packages/SQLitePCLRaw.provider.winsqlite3) | The `winsqlite3` provider is for Windows 10 environments. |
+
+To use the `sqlite3` provider use the following commands:
+
+### [.NET Core CLI](#tab/netcore-cli)
+
+```dotnetcli
+dotnet add package Microsoft.Data.Sqlite.Core
+dotnet add package SQLitePCLRaw.core
+dotnet add package SQLitePCLRaw.provider.sqlite3
+```
+
+### [Visual Studio](#tab/visual-studio)
+
+``` PowerShell
+Install-Package Microsoft.Data.Sqlite.Core
+Install-Package SQLitePCLRaw.core
+Install-Package SQLitePCLRaw.provider.sqlite3
+```
+
+---
+
+With the packages installed, you then set the provider to the `sqlite3` instance.
+
+[!code-csharp[](../../../../samples/snippets/standard/data/sqlite/SqliteProviderSample/Program.cs)]
+
+## SQLitePCLRaw custom providers
 
 You can use your own build of SQLite by leveraging the `SQLitePCLRaw.provider.dynamic_cdecl` package. In this case, you're responsible for deploying the native library with your app. Note, the details of deploying native libraries with your app vary considerably depending on which .NET platform and runtime you're using.
 
-First, you'll need to implement IGetFunctionPointer. The implementation is fairly trivial on .NET Core.
```

Comment:
lol, the message between the lines was that this is ridiculously hard on .NET Framework 😏

Reply:
Also, IGetFunctionPointer should be a link or `code`

Reply:
@Thraka - yes, that was already part of these changes https://github.com/dotnet/docs/pull/18437/files#diff-e6f920f06dea9338118692297db0e8c1R82